### DISCUSSION TITLE
Is there anything like the Schema Transfer Wizard from MySQL Workbench?

### DISCUSSION THREAD

**Discussion by ghost** [dbeaver/dbeaver#18546] [Ideas]
With the Schema Transfer Wizard from MySQL Workbench, you just pick a source and target DB and schema and it will automatically create the same schema on the target server and import all the table data.

Is there feature that is as streamlined as this, for DBeaver?

**Comment by LonwoLonwo** [comment]
You can use [Schema Compare](https://github.com/dbeaver/dbeaver/wiki/Schema-compare) with migration and then [Data Migration](https://github.com/dbeaver/dbeaver/wiki/Data-migration).

Or just [Data Migration](https://github.com/dbeaver/dbeaver/wiki/Data-migration), but it will move only tables (with data), not other objects like view, procedure, etc.
